homeassistant/components/tessie/entity.py

```diff
@@ -65,8 +65,12 @@ async def run(
         except ClientResponseError as e:
             raise HomeAssistantError from e
         if response["result"] is False:
+            name: str = getattr(self, "name", self.entity_id)
             raise HomeAssistantError(
-                response.get("reason", "An unknown issue occurred")
```

Comment:
Is there a way to keep the handle *unknown issues* part?

Reply:
Yes, but as far as I know the API never returns false without a reason.

Reply:
Added `reason: str = response.get("reason", "unknown")`